Sources/Hyperspace/Service/Network/NetworkActivityIndicatable.swift

```diff
@@ -74,11 +78,15 @@ private extension NetworkActivityController {
         delayedHide?.cancel()
         delayedHide = nil
         
+        #if MACOS
```

Comment:
Same as above - the protocol is abstracting the use of UIApplication. The isNetworkActivityIndicatorVisible call could be forwarded to a custom indicator (as it would need to be on macOS).